docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -77,12 +77,12 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
     transpiled_circuit = cloud_transpiler_service.run(circuit)
```

Comment:
Consider adding a comment explaining the AI-powered transpiling passes decision logic for better documentation clarity